content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Octopus Deploy
+
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 1
```

Comment:
Is this weight correct? Are all CI/CD docs given the same weight, and then sorted alphabetically? Or do we manually set the weights so that the pages are in some fixed, ordering?

Reply:
Yes, all of the docs are given a weight of 1 except for the `Other` doc, which talks about a yet-to-be added CI doc.